src/components/TravelSection.tsx

```diff
@@ -59,31 +59,79 @@ const TravelSection = () => {
     window.open(link, '_blank', 'noopener,noreferrer');
   };
 
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'ArrowLeft') {
+      event.preventDefault();
+      prevTravel();
+    } else if (event.key === 'ArrowRight') {
+      event.preventDefault();
+      nextTravel();
+    }
+  };
+
+  const formatPrice = (price: number) => {
+    return price.toLocaleString('ko-KR');
+  };
+
   return (
-    <div className={styles.travelSection}>
-      <button className={`${styles.navButton} ${styles.navButtonPrev}`} onClick={prevTravel}>
-        <img src={chevronLeft} className={styles.navButtonIcon} />
+    <div
+      className={styles.travelSection}
+      role="region"
+      aria-label="지금 떠나기 좋은 여행 상품 캐로셀"
+      onKeyDown={handleKeyDown}
+      tabIndex={0}
+    >
+      <button
+        className={`${styles.navButton} ${styles.navButtonPrev}`}
+        onClick={prevTravel}
+        type="button"
+      >
+        <img src={chevronLeft} className={styles.navButtonIcon} alt="이전 여행 상품" />
       </button>
-      <div className={styles.carousel}>
+
+      <div className={styles.carousel} role="group" aria-live="polite">
```

Comment:
```suggestion
      <fieldset className={styles.carousel} role="group" aria-live="polite">
```
`fieldset` 태그에 `role="group"` 속성이 내부적으로 포함되어 있어서 `fieldset` 태그를 적용할 수 있는지 확인해보셔도 좋을 것 같습니다🙃

Reply:
오 fieldset이란 태그도 있군요!! 찾아보니 form이나 input에 많이 쓰이는 태그인것 같아요
캐러셀에 적용해보는건 고민을 해봐야될 것 같습니다!